src/commands/plugins.rs

```diff
@@ -416,6 +420,38 @@ impl Upgrade {
     }
 }
 
+#[derive(Parser, Debug)]
+pub struct Inspect {
+    /// Name of Spin plugin.
+    pub name: String,
+}
+
+impl Inspect {
+    pub async fn run(self) -> Result<()> {
+        let manager = PluginManager::try_default()?;
+        let manifest = manager
+            .get_manifest(
+                &ManifestLocation::PluginsRepository(PluginLookup::new(&self.name, None)),
+                false,
+                SPIN_VERSION,
+            )
+            .await?;
+
+        println!(
+            "{}: {} (License: {})\n{}\n\n{}",
+            manifest.name(),
+            manifest.version(),
+            manifest.license(),
+            manifest
+                .homepage_url()
+                .map(|u| u.to_string())
+                .unwrap_or_default(),
```

Comment:
Nit: This will leave an empty line if there's no homepage.  You could workaround this by moving the `\n` into the `map` a la `manifest.homepage_url().map(|u| format!("{u}\n")).unwrap_or_default()` I think.  It's not very important though.

Reply:
updated the output accordingly